bundles/org.openhab.binding.chromecast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -244,7 +244,7 @@
 		<item-type>DateTime</item-type>
 		<label>Broadcast Date</label>
 		<description>The broadcast date of the currently playing media</description>
-		<category>time</category>
+		<category>Calendar</category>
```

Comment:
`Calendar` is not listed as a valid channel category in the documentation: https://www.openhab.org/docs/concepts/categories.html#channel-categories
Searching this channel category in all bindings, it is not found.
In case it is really a valid category, at least the documentation should be updated.

Reply:
Icon is available in the classic iconset: https://github.com/openhab/openhab-webui/blob/main/bundles/org.openhab.ui.iconset.classic/src/main/resources/icons/calendar.png
Documentation is just outdated !
So using it as channel category is I guess valid.